source/_integrations/litterrobot.markdown

```diff
@@ -2,7 +2,7 @@
 title: Litter-Robot
 description: Instructions on how to integrate a Litter-Robot Wi-Fi-enabled, automatic, self-cleaning litter box to Home Assistant.
 ha_category:
-  - Vacuum
+  - Pets
```

Comment:
We don't have such a category.

Reply:
Doesn't the website just generate the list of categories by what is in ha_category? For example, MQTT has "Tag Scanner" and AfterShip and 17Track have "Postal Service" as a ha_category.